src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension/src/extension.ts

```diff
@@ -0,0 +1,72 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+
+import * as vscode from 'vscode';
+import { LocalDebugProxyManager } from './LocalDebugProxyManager';
+import { spawn } from 'child_process';
+
+let launchDebugProxy: vscode.Disposable;
+let killDebugProxy: vscode.Disposable;
+
+export function activate(context: vscode.ExtensionContext) {
+    const outputChannel = vscode.window.createOutputChannel("Blazor WASM Debug Proxy");
+    const pidsByUrl = new Map<string, number>();
+
+    launchDebugProxy = vscode.commands.registerCommand('ms-blazorwasm-companion.launchDebugProxy', async (version = "5.0.0") => {
```

Comment:
We only expect the Razor extension to call this so I suppose it's not necessary to default here (after all, it's redundant since we default `debugProxyVersion` in the package config).

> Asking because if we truly can only understand "5.0.0" or the expectation is to download a new "companion" then I'd imagine it'd make more sense to hardcode

We can understand all debug proxy versions. I used 5.0.0 because it's the most recent, verified major release. Someone could use a package version like `5.0.0-rc2` or `6.0.0-preview1` and things should the same with this.